evap/evaluation/models.py

```diff
@@ -159,6 +159,26 @@ class Meta:
     def __str__(self):
         return self.name
 
+    def can_delete(self):
+        return not self.courses.all().exists()
```

Comment:
on all other models this is called `can_staff_delete`, could you use that here as well?